web/src/scenes/team/team.tsx

```diff
@@ -1,5 +1,25 @@
-import { Fragment } from "react";
+import { AuthRequired } from "@/components/AuthRequired";
+import { Layout } from "@/components/Layout";
+import { memo, useEffect } from "react";
+import { Details } from "./Details";
+import { Header } from "./Header";
+import { MemberList } from "./MemberList";
+import { getTeamStore, useTeamStore } from "@/stores/teamStore";
 
-export function Team(): JSX.Element | null {
-  return <Fragment>I am not yet implemented.</Fragment>;
-}
+export const Team = memo(function Team() {
+  const { team, fetchTeam } = useTeamStore(getTeamStore);
+
+  useEffect(() => {
+    fetchTeam();
+  }, []);
+
+  return (
+    <AuthRequired>
+      <Layout mainClassName="grid gap-y-8">
+        <Header />
+        <Details />
```

Comment:
I think this way it looks more structured. Besides, in the future, we need to fill in name input from the store and then handle it. In my opinion, it's better if handlers will be placed in the separate Details component.

Reply:
@paolodamico What do you think? Maybe it's better to have small components inline, not to navigate through the IDE, trying to find this component, so it can be more clearly